website/src/components/demos/geo-layer-demos.js

```diff
@@ -0,0 +1,89 @@
+/* global fetch */
+import {VectorTile} from '@mapbox/vector-tile';
+import Protobuf from 'pbf';
+import createLayerDemoClass from './layer-demo-base';
+import {DATA_URI} from '../../constants/defaults';
+
+import {
+  GreatCircleLayer,
+  S2Layer,
+  TileLayer
+} from 'deck.gl';
+
+export const GreatCircleLayerDemo = createLayerDemoClass({
+  Layer: GreatCircleLayer,
+  dataUrl: `${DATA_URI}/bart-segments.json`,
```

Comment:
Let's use some dataset of big world cities. The layer does not make any sense on a local scale.